helpers/sri_helper.py

```diff
@@ -0,0 +1,280 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=too-many-lines
+import json
+import re
+import urllib
+import urllib.parse
+from helpers.data_helper import append_domain_entry
+from helpers.setting_helper import get_config
+from models import Rating
+
+# pylint: disable=too-many-arguments
+def rate_sri(result_dict, global_translation, local_translation,
+             org_domain, org_www_domain, domain):
+    """
+    This function rates the Subresource Integrity (SRI) of a given domain.
+
+    Parameters:
+    result_dict (dict): A dictionary containing the results of the SRI checks.
+    global_translation (function): A function to translate text to a global language.
+    local_translation (function): A function to translate text to a local language.
+    org_domain (str): The original domain.
+    org_www_domain (str): The original domain with 'www.' prefix.
+    domain (str): The domain to be rated.
+
+    Returns:
+    Rating: A Rating object containing the overall rating,
+            standards rating, and integrity and security rating.
+    """
+    rating = Rating(global_translation, get_config('general.review.improve-only'))
+    if not isinstance(result_dict[domain], dict):
+        return rating
+
+    if domain not in (org_domain, org_www_domain):
+        return rating
+
+    if 'SRI-WITH-ERRORS' in result_dict[domain]['features']:
```

Comment:
@cockroacher We should write out the errors we encountered, fixing it otherwise are almost impossible + we should be helping not just grading.